versions.lock

```diff
@@ -424,19 +424,26 @@ org.glassfish.grizzly:grizzly-framework:2.4.4 (1 constraints: 670fe271)
 org.glassfish.grizzly:grizzly-http:2.4.4 (1 constraints: 2b127cf5)
 org.glassfish.grizzly:grizzly-http-server:2.4.4 (2 constraints: bb2c391b)
 org.glassfish.grizzly:grizzly-http-servlet:2.4.4 (1 constraints: 681bda3f)
-org.glassfish.jersey.containers:jersey-container-grizzly2-http:2.35 (2 constraints: 623ed2c2)
-org.glassfish.jersey.containers:jersey-container-grizzly2-servlet:2.35 (1 constraints: 292310ba)
-org.glassfish.jersey.containers:jersey-container-servlet:2.35 (1 constraints: 3a1b7724)
-org.glassfish.jersey.containers:jersey-container-servlet-core:2.35 (2 constraints: a031bd78)
-org.glassfish.jersey.media:jersey-media-jaxb:2.35 (1 constraints: e11958ca)
-org.glassfish.jersey.test-framework:jersey-test-framework-core:2.35 (2 constraints: 062875af)
-org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-grizzly2:2.35 (1 constraints: de04fe30)
+org.glassfish.jersey.containers:jersey-container-grizzly2-http:2.39.1 (2 constraints: 283f089c)
+org.glassfish.jersey.containers:jersey-container-grizzly2-servlet:2.39.1 (1 constraints: 8c230a01)
+org.glassfish.jersey.containers:jersey-container-servlet:2.39.1 (1 constraints: 9d1b845b)
+org.glassfish.jersey.containers:jersey-container-servlet-core:2.39.1 (2 constraints: 66329628)
+org.glassfish.jersey.media:jersey-media-jaxb:2.39.1 (1 constraints: 441ab3fe)
+org.glassfish.jersey.test-framework:jersey-test-framework-core:2.39.1 (2 constraints: cc283b2f)
+org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-grizzly2:2.39.1 (1 constraints: 4105533b)
 org.hdrhistogram:HdrHistogram:2.1.12 (1 constraints: 520d2029)
 org.hsqldb:hsqldb:2.7.1 (1 constraints: 0c050b36)
 org.jetbrains.kotlin:kotlin-reflect:1.6.10 (2 constraints: 6224e0e2)
+org.junit.jupiter:junit-jupiter:5.9.1 (3 constraints: 844699a3)
+org.junit.jupiter:junit-jupiter-api:5.9.1 (4 constraints: 8b386434)
+org.junit.jupiter:junit-jupiter-engine:5.9.1 (2 constraints: 2117d63c)
+org.junit.jupiter:junit-jupiter-params:5.9.1 (2 constraints: 2117d63c)
+org.junit.platform:junit-platform-commons:1.9.1 (3 constraints: ee29f32b)
+org.junit.platform:junit-platform-engine:1.9.1 (2 constraints: bc19e1f3)
 org.latencyutils:LatencyUtils:2.0.3 (1 constraints: 210dcd1b)
 org.mockito:mockito-core:3.8.0 (1 constraints: 0d051236)
 org.objenesis:objenesis:3.1 (1 constraints: b00a12bd)
+org.opentest4j:opentest4j:1.2.0 (2 constraints: cd205b49)
```

Comment:
This is a new dependency that needs license or to be excluded?

Reply:
I checked it out, it is JUnit5 that has spun off some exception classes into a new project, hoping that other test frameworks will also standardize on it. I think we need to add it, but it is super tiny. Will add a commit.

Reply:
Pushed the license file